### ISSUE TITLE
Highlight photos of people who belong to LACEN

### ISSUE THREAD

**Issue by abhi18av** [REVIGET/REVIGET-website#2]
This modification could be in the form of a logo within a photo or just a green border in different photos.

**Comment by Mxrcon** [comment]
Hmm, perhaps we could have:

## "LACEN-PA"

## "LACEN-CE"

And the background color, we could modify the base sccs, for using a greenish background within a custom background class as used for PI in masiello website
 ```
.black-background {
  background-color: black;
}

.primary-background {
  background-color: $primary;
}

.gold-background {
  background-color: $gold;
}

.sand-background {
  background-color: $sand;
}
```

This is how background is called into the masiello `people/index.qmd`
```
:::{.sand-background}
## Alumni



::: {#alumni}
```` 
What do you think?

**Comment by abhi18av** [comment]
@Mxrcon , good idea but tangential to our use-case since we need to highlight the individual photo and not the entire section.


This is a good reference https://usage.imagemagick.org/crop/#border - I'm pretty sure that nodejs world should have a decent automation for this kind of work.
